staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/types.go

```diff
@@ -48,9 +48,13 @@ type ListOptions struct {
 	// Defaults to changes from the beginning of history.
 	// When specified for list:
 	// - if unset, then the result is returned from remote storage based on quorum-read flag;
-	// - if it's 0, then we simply return what we currently have in cache, no guarantee;
-	// - if set to non zero, then the result is at least as fresh as given rv.
+	// - if it's 0, the result may contain arbitrarily old data, no guarantee; Only valid with ResourceVersionMatch of “Minimum” (or “Legacy” acting as “Minimum”).
+	// - if set to non zero, ResourceVersionMatch applies.
 	ResourceVersion string
+	// ResourceVersionMatch specifies how ResourceVersion parameter is applied.
+	// Defaults to ResourceVersionMatchLegacy.
```

Comment:
nit: ResourceVersionMatchLegacy is not defined anywhere (I'm assuming that ResourceVersionMatchLegacy should be defined to empty string, right)?

Reply:
I dropped `ResourceVersionMatchLegacy` entirely but missed this comment. I'll clean it up.